func/code.func

```diff
@@ -40,7 +43,7 @@ _ pack_state(int o_seqno, int n_wins, int in_amount, int out_amount, int hash, s
       begin_cell()                ;; SECOND CELL:
         .store_slice(owner1)      ;; 267 bit | owner 1, @sergei_ton
```

Comment:
I think only one owner should be left; if needed, it may be a contract forwarding winnings to other wallets.

Reply:
You can use any owner address during development, of course; no need to include splitter contract in this repository.